### PR TITLE
Fixes post author in layout

### PR DESCRIPTION
**Pull request by rdev5**

### CODE REVIEW

**Comment by victor** [comment]
@davidtmiller  please merge this

**Comment by davidtmiller** [comment]
Good fix thank you.

File: _layouts/post.html [hunk 1]
```diff
@@ -5,21 +5,23 @@
...
-  <div class="container">
-    <div class="row">
-      <div class="col-lg-8 col-md-10 mx-auto">
-        <div class="post-heading">
-          <h1>{{ page.title }}</h1>
-          {% if page.subtitle %}
-          <h2 class="subheading">{{ page.subtitle }}</h2>
-          {% endif %}
-          <span class="meta">Posted by
-            <a href="#">{% if page.author %}{{ page.author }}{% else %}{{ site.author }}{% endif %}</a>
```
Comment (philtgun) [inline _layouts/post.html]: This post author fix was not properly merged, can it be fixed?

Comment (davidtmiller) [reply _layouts/post.html]: I've fixed this now in `master` not sure what happened here...

**Comment by davidtmiller** [comment]
Should be fixed!

**Pull request merged**